### PR TITLE
ACS-8315: avatar component

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

- simple `Avatar` component (`<adf-avatar>`)
- theming and usage docs

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://hyland.atlassian.net/browse/ACS-8315

### CODE REVIEW

File: lib/core/src/lib/avatar/avatar.component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    @Input()
+    initials: string = 'U';
+
+    @Input()
+    tooltip: string = '';
+
+    @HostBinding('style.--adf-avatar-size')
+    @Input()
+    size = getComputedStyle(document.documentElement).getPropertyValue('--adf-avatar-size');
```
Comment (BSekula) [inline lib/core/src/lib/avatar/avatar.component.ts:41]: Is the size used anywhere? 🤔

**Comment by BSekula** [comment]
Can we add some small test here?

File: lib/core/src/lib/avatar/avatar.component.ts (hunk 1 above)
Comment (DenysVuika) [reply lib/core/src/lib/avatar/avatar.component.ts:41]: see the demo (demo shell) in the same PR... using both size and cursor do introduce the dropdown support

Comment (DenysVuika) [reply lib/core/src/lib/avatar/avatar.component.ts:41]: @BSekula that's the way of using either custom value coming from the `Input`, or rolling back to the value coming from some external CSS

**Review by MichalKinas — APPROVED** [review]

**Review by g-jaskowski — APPROVED** [review]

**Pull request merged**